corehq/apps/locations/bulk_management.py

```diff
@@ -217,14 +217,15 @@ def lookup_old_collection_data(self, old_collection, locs_by_code):
         )
         if self.parent_code and self.parent_code != ROOT_LOCATION_TYPE:
             try:
-                self._new_parent_stub = locs_by_code[self.parent_code]
+                self.new_parent_stub = locs_by_code[self.parent_code]
             except KeyError:
                 # TODO require all referenced locations to be in stubs and then
                 # remove this exception handler and fake stub nonsense.
                 # Breaks test: TestBulkManagement.test_large_upload
                 class fake_stub:
                     db_object = old_collection.locations_by_site_code[self.parent_code]
-                self._new_parent_stub = fake_stub
+                    do_delete = False
```

Comment:
I don't think this is referenced anywhere - should this be
`self.new_parent_stub.do_delete`?

Reply:
I mean the line
```python
do_delete = False
```
`do_delete` is a local variable that's not referenced anywhere.  Should it be `self.do_delete`?

Reply:
it's a class attribute, just like `db_object` above it

Reply:
Oh wow - yup.  Carry on.